test/run_test.py

```diff
@@ -276,6 +276,16 @@ def parse_args():
         choices=TESTS,
         metavar='TESTS',
         help='select the last test to run (excludes following tests)')
+    parser.add_argument(
+        '-b',
```

Comment:
nit: I probably woudn't give this flag a short form... and if I did it wouldn't be `-b`